### DISCUSSION TITLE
Welcome to UltraLiteDB Discussions!

### DISCUSSION THREAD

**Discussion by rejemy** [rejemy/UltraLiteDB#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dangershony** [comment]
How active will the dev/devs be on this repo?
To avoid situation like the critical bug in litedb that was not addressed for many month (and I think still not totally solved).

**Reply by rejemy** [reply]
It is not very active. The library is mainly used by me for an internal project I'm working on. I do fix any bugs I run into, and am happy to look at any other bugs anyone else finds, because I want this thing to be solid, since I'm shipping my own software on it. But yes, to be honest this is not an active repo.

**Reply by dangershony** [reply]
ok thanks fair enough, we may need a k/v db for unity on our project, when that happens we'll probably use UltraLightDB and will be more active here.
